runtime/lib/ttnn/operations/CMakeLists.txt

```diff
@@ -31,7 +31,7 @@ add_library(TTRuntimeTTNNOps
 )
 
 set_property(TARGET TTRuntimeTTNNOps PROPERTY CXX_STANDARD 20)
-target_compile_options(TTRuntimeTTNNOps PUBLIC -mavx -mavx2)
+target_compile_options(TTRuntimeTTNNOps PUBLIC -mavx -mavx2 -fsized-deallocation)
```

Comment:
sized-deallocations are needed because we are pulling in some headers which use custom allocators....